### PR TITLE
refactor: concise the return of bool expression

### PR DESCRIPTION
**Pull request by QianqianLu-WebDev**
#### What type of PR is this?
/kind cleanup

#### What this PR does / why we need it:
clean code

#### Does this PR introduce a user-facing change?
NONE

do-not-merge/release-note-label-needed

### CODE REVIEW

File: pkg/controller/podgc/gc_controller_test.go [hunk 1]
```diff
@@ -57,10 +57,7 @@ func compareStringSetToList(set sets.String, list []string) bool {
 			return false
 		}
 	}
-	if len(list) != len(set) {
```
Comment (kerthcet) [inline pkg/controller/podgc/gc_controller_test.go:60]: Maybe move this to the head of the function, it's more effective.

Comment (QianqianLu-WebDev) [reply pkg/controller/podgc/gc_controller_test.go:60]: Exactly. Thx.

**Comment by dims** [comment]
If you still need this PR then please rebase, if not, please close the PR

**Pull request closed without merging**